packages/source-contentful/index.js

```diff
@@ -80,6 +80,9 @@ class ContentfulSource {
           )
         } else if (typeof value === 'object' && typeof value.sys !== 'undefined') {
           fields[key] = this.createReferenceField(value)
+        } else if (typeof value === 'object' && value.nodeType === 'document') {
+          // value is Rich Text
+          fields[key] = JSON.stringify(value)
```

Comment:
```suggestion
          fields[key] = documentToHtmlString(value)
```
Would it be possible to generate the HTML here instead?

Reply:
It's possible, but because the `documentToHtmlString` method takes a config option, we would have to pass that in through `gridsome.config.js` and then it becomes less customizable if, for instance, you needed to render custom elements for different pages. Like how in the readme we pass a custom config for handling embedded image assets

Reply:
I understand :) Maybe the rich text fields could contain two properties: `document` and `html`? The `html` property could be used for people who just want to show the HTML.